packages/ic-response-verification/tests/v2_response_verification_certification_scenarios.rs

```diff
@@ -183,6 +183,7 @@ mod tests {
             result,
             VerificationResult::Failed {
                 verification_version,
+                reason: _,
```

Comment:
Could we add the expected reason as a parameter to the tests? Or do you think it would be overkill?

Reply:
All errors here are `InvalidExpressionPath`, so, initially i thought it would be an overkill to repeat this for each test case, but since it's not a big effort i've just added either way, it will help us in the future if any of those case scenarios deviate from the expect error.